packages/teleport/src/DesktopSession/TopBar.tsx

```diff
@@ -79,6 +103,12 @@ const StyledClipboard = styled(Clipboard)`
   align-self: 'center';
 `;
 
+const StyledFolderShared = styled(FolderShared)`
+  font-weight: ${({ theme }) => theme.fontWeights.bold};
+  font-size: ${({ theme }) => theme.fontSizes[6] + 'px'};
```

Comment:
i like this, we should be doing this more to keep in line with our theme @rudream